README.md

```diff
@@ -527,18 +534,16 @@ An application that contains this engine must explicitly fulfill these
 dependencies. For example:
 
 ```js
-import Ember from 'ember';
-import Resolver from 'ember-resolver';
-import loadInitializers from 'ember/load-initializers';
+import Application from '@ember/application';
+import Resolver from './resolver';
+import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
 
-Ember.MODEL_FACTORY_INJECTIONS = true;
-
 const { modulePrefix, podModulePrefix } = config;
 
-const App = Ember.Application.extend({
-  modulePrefix,
-  podModulePrefix,
+const App = Application.extend({
+  modulePrefix: config.modulePrefix,
+  podModulePrefix: config.podModulePrefix,
```

Comment:
because of  line 542 you don't need to change these two preceding lines

Reply:
You are wrong. This is the actual blueprint of Ember's latest versions without the use of destructuring, and the new modules API. 

https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/app/app.js

Reply:
Thanks, i fixed